docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: kyma alpha install
+title: kyma alpha deploy
```

Comment:
Please also rename the package (`cmd/alpha/install` -> `cmd/alpha/deploy`) so that it is consistent with all other commands and avoid confusion.